eng/Versions.props

```diff
@@ -38,7 +38,7 @@
     <SystemReflectionMetadataLoadContextVersion>7.0.0-alpha.1.21614.10</SystemReflectionMetadataLoadContextVersion>
     <DeploymentReleasesVersion>1.0.0-preview1.1.21112.1</DeploymentReleasesVersion>
     <SystemManagementPackageVersion>4.6.0</SystemManagementPackageVersion>
-    <SystemCommandLineVersion>2.0.0-beta1.21601.2</SystemCommandLineVersion>
+    <SystemCommandLineVersion>2.0.0-beta2.21616.4</SystemCommandLineVersion>
```

Comment:
When updating this, the Versions.Details.xml should be updated as well (e.g. https://github.com/dotnet/sdk/pull/23032) otherwise there is a good changes source-build will be broken.

Reply:
Thanks @MichaelSimons! Follow up - are there docs I can read on this whole process (assume I know zero about source-build), and is there tooling we can put in the repo to ensure they are kept in sync (or better - automatically updated)?

Reply:
Usually both these files are updated via automations (note that this thing is under an ItemGroup labeled "Automated"). But I think System.CommandLine is a bit of an exception given how dramatically their APIs change between releases.